vendor/assets/javascripts/es5-shim.js

```diff
@@ -0,0 +1,1572 @@
+/*!
```

Comment:
I think that only worked for the javascript specs and not the ruby specs.  So I guess it's more complicated than just only sourcing it in the js specs.  I'll see if I can find a cleaner solution.